db/migrate/20210428101853_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.string :title
+      t.text :description
+      t.float :price
+
```

Comment:
you can also add `null` constraints when creating tables if needed like this:

`t.string :title, null: false`